packages/custom/orders/server/routes/orders.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var orders = require('../controllers/orders');
+
+// Order authorization helpers
```

Comment:
This should be placed in get and update methode for order so that user 1 cannot view or update user 2's order. Refer Carts service for the same